Sources/JWTKit/Utilities/BigNumber.swift

```diff
@@ -16,8 +16,8 @@ class BigNumber {
         CJWTKitBoringSSL_BN_free(self.c);
     }
 
-    public static func convert(_ bnBase64: String) -> BigNumber? {
-        guard let data = Data(base64Encoded: bnBase64) else {
+    public static func convert(_ bnBase64Url: String) -> BigNumber? {
```

Comment:
As below - this is technically non-breaking but I'd prefer to create a new function that is explicitly for Base64URL strings to avoid breaking stuff people are using this for and switching our ECDSA implementation over to that

Reply:
Can I propose
```swift
public convenience init?(base64URL: String)
```
for the `BigNumber` class?

Reply:
Yep that sounds good